src/components/Select/_Select.scss

```diff
@@ -2,47 +2,62 @@
 .select {
   @include space-stack-xs;
 
-  -moz-appearance: none;
-  -webkit-appearance: none;
-  appearance: none;
-  background-color: var(--ui-white);
-  /* Arrow svg */
-  /* More info here: https://css-tricks.com/probably-dont-base64-svg/ */
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='currentColor' viewBox='0 0 24 24'%3E%3Cpath d='M10.526 12.871L.263 1.676 1.737.324 12 11.52 22.263.324l1.474 1.352L13.474 12.87a2 2 0 01-2.948 0z'/%3E%3C/svg%3E");
-  background-position: right 0.7em top 70%;
-  background-repeat: no-repeat;
-  background-size: var(--space) auto;
-  border: 1px solid var(--ui-gray-medium);
-  border-radius: 2px;
-  display: block;
-  font-size: 1rem;
-  padding: var(--space-xs) var(--space-xl) var(--space-xs) var(--space-s);
-  width: 100%;
+  .selectfield {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    background-color: var(--ui-white);
+    /* Arrow svg */
+    /* More info here: https://css-tricks.com/probably-dont-base64-svg/ */
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='currentColor' viewBox='0 0 24 24'%3E%3Cpath d='M10.526 12.871L.263 1.676 1.737.324 12 11.52 22.263.324l1.474 1.352L13.474 12.87a2 2 0 01-2.948 0z'/%3E%3C/svg%3E");
```

Comment:
This is neat but mentions issues with some browsers that may no longer be a problem. Is there a benefit from doing this rather than rendering an `Icon` component?

Reply:
@EdwinGuzman This was how the component was initially put together.  Let me see what I can do.

Reply:
@EdwinGuzman Replacing the arrow on a `select` input is traditionally done with CSS.  I think refactoring the code to use the Icon component would be overkill.  That being said, having to copy and paste the `svg` text right into the CSS is burdensome.  In a webpack/react setup, how would I directly reference our arrow `svg`?

Reply:
Ah gotcha. Let's leave this like this. It may be more complicated to get webpack involved than doing what's already here.